packages/container/tests/container_test.js

```diff
@@ -519,6 +519,16 @@ QUnit.test('Lazy injection validations are cached', function() {
   container.lookup('apple:main');
 });
 
+QUnit.test('A registered factory is returned from lookupFactory', function() {
```

Comment:
This test title should reference `ownerInjection`. Perhaps "An object with its owner pre-set should be returned from `ownerInjection`".